src/components/Nav.js

```diff
@@ -1,9 +1,15 @@
+import { useEffect, useState } from 'react';
 import { useNavigate, useLocation } from 'react-router-dom';
 import styled, { css } from 'styled-components';
 
 const Nav = () => {
   const navigate = useNavigate();
   const location = useLocation();
+  const [isLogin, setIsLogin] = useState(false);
+
+  useEffect(() => {
+    setIsLogin(localStorage.getItem('access_token') && true);
+  }, []);
```

Comment:
- 이 값이 state일 필요가 있을까요?
```suggestion
  useEffect(() => {
    setIsLogin(!!localStorage.getItem('access_token'));
  }, []);
```
- truthy, falsy 값을 boolean 형태로 변환하려면 not 연산자를 두번 사용해주면 됩니다!

Reply:
감사합니다!